src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/ResendStaffNotificationsCoreCaseDataTest.java

```diff
@@ -0,0 +1,248 @@
+            )
+        ).willReturn(listOfCaseDetails());
+
+        makeRequest(claim.getReferenceNumber(), event).andExpect(status().isOk());
+
+        verify(emailService, times(2)).sendEmail(eq("sender@example.com"), emailDataArgument.capture());
+
+        EmailData emailData = emailDataArgument.getValue();
+        assertThat(emailData.getTo()).isEqualTo("recipient@example.com");
+        assertThat(emailData.getSubject()).isEqualTo("Claim " + claim.getReferenceNumber() + " issued");
+        assertThat(emailData.getMessage()).isEqualTo("Please find attached claim.");
+    }
+
+    @Test
+    public void shouldRespond409AndNotProceedForMoreTimeRequestedEventWhenMoreTimeNotRequested() throws Exception {
+        final String event = "more-time-requested";
+
+        given(coreCaseDataApi.searchForCitizen(
+            any(),
+            any(),
+            any(),
+            any(),
+            any(),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE, "sortDirection", "desc"))
+            )
+        ).willReturn(listOfCaseDetails());
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isConflict());
+
+        verify(notificationClient, never()).sendEmail(any(), any(), any(), any());
+    }
+
+    @Test
+    public void shouldRespond200AndSendNotificationsForMoreTimeRequestedEvent() throws Exception {
+        final String event = "more-time-requested";
+
+        given(coreCaseDataApi.searchForCitizen(
+            any(),
+            any(),
+            any(),
+            any(),
+            any(),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE, "sortDirection", "desc"))
+            )
+        ).willReturn(listOfCaseDetailsWithDefendant());
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isOk());
+
+        verify(notificationClient).sendEmail(eq("staff-more-time-requested-template"),
+            eq("recipient@example.com"),
+            any(),
+            eq("more-time-requested-notification-to-staff-" + CASE_REFERENCE));
+    }
+
+    @Test
+    public void shouldRespond409AndNotProceedForResponseSubmittedEventWhenResponseNotSubmitted() throws Exception {
+        final String event = "response-submitted";
+
+        given(coreCaseDataApi.searchForCitizen(
+            any(),
+            any(),
+            any(),
+            any(),
+            any(),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE, "sortDirection", "desc"))
+            )
+        ).willReturn(listOfCaseDetails());
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isConflict());
+
+        verify(emailService, never()).sendEmail(any(), any());
+    }
+
+    @Test
+    public void shouldRespond200AndSendNotificationsForResponseSubmittedEvent() throws Exception {
+        final String event = "response-submitted";
+
+        given(coreCaseDataApi.searchForCitizen(
+            eq(AUTHORISATION_TOKEN),
```

Comment:
Please make code consistent, in other places you use `any()` to ignore less relevant arguments to search method.